crates/volta-core/src/tool/package/metadata.rs

```diff
@@ -13,7 +13,7 @@ use semver::Version;
 
 /// Configuration information about an installed package
 ///
-/// Will be stored in <VOLTA_HOME>/tools/user/packages/<package>.json
+/// Will be stored in <VOLTA_HOME>/tools/user/packages/\<package>.json
```

Comment:
Is this added `\` intentional?

Reply:
It needs to be escaped, otherwise:
![image](https://user-images.githubusercontent.com/5764917/185100107-c637853f-1319-4daa-b0c4-dc65fed36820.png)

I updated it to use inline code block syntax.